docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -209,8 +213,26 @@ Release
       git tag --annotate "${TAG}" "${BRANCH}" -m "Tag version ${TAG}"
       git push origin "${TAG}"
 
+#. Go to `Jenkins`_ and force a build on the release branch to test the latest commit.
+
+   Currently, jobs cannot be created for git tags so the latest commit must be test instead.
+   This must be the same commit as the tag.
+   The git commit that was used can be seen on the summary page for any build.
+
+   If `FLOC-3917 <https://clusterhq.atlassian.net/browse/FLOC-3917>`_ has not been resolved, then the "_build_vagrant_basebox_for_flocker_tutorial" job must also be run.
+
+   If `FLOC-3916 <https://clusterhq.atlassian.net/browse/FLOC-3916>`_ has not been resolved, then after the previous step, the Vagrant box artifacts must be moved within the S3 bucket.
+
```

Comment:
Remove the blank line, so that the next instruction clearly relates to the FLOC-3916 issue not being resolved.